catroid/src/org/catrobat/catroid/ui/fragment/FormulaEditorListFragment.java

```diff
@@ -133,7 +145,14 @@ public void onCreate(Bundle savedInstanceState) {
 		} else if (tag.equals(LOGIC_TAG)) {
 			itemsIds = LOGIC_ITEMS;
 		} else if (tag.equals(SENSOR_TAG)) {
-			itemsIds = SENSOR_ITEMS;
+			//TODO ALbert:
```

Comment:
There is no hurry, currently the release is being prepared.
Btw please write in english.
Regards